src/prefect/orion/models/task_runs.py

```diff
@@ -76,11 +76,13 @@ async def create_task_run(
     model = result.scalar()
 
     if model.created == now and task_run.state:
+        # creating a run should never involve concurrency limits, so we use a NoOp policy
         await models.task_runs.set_task_run_state(
             session=session,
             task_run_id=model.id,
             state=task_run.state,
             force=True,
+            task_policy=NoOpTaskPolicy(),
```

Comment:
After a second look, I don't think we can do this safely. There are a few rules that seem like they need to run here https://github.com/PrefectHQ/prefect/blob/main/src/prefect/orion/orchestration/global_policy.py#L21